android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -19,10 +19,10 @@
 #define FUNC(RETURN_TYPE, NAME, ...)                                      \
   extern "C" {                                                            \
   JNIEXPORT RETURN_TYPE Java_org_aomedia_avif_android_AvifDecoder_##NAME( \
-      JNIEnv* env, jobject thiz, ##__VA_ARGS__);                          \
+      __VA_ARGS__);                                                       \
```

Comment:
hmm, i don't like the idea of repeating the jni env parameters in all the functions. it pushes the actual argument list to the back and it's a mix of named and unnamed parameters.

how about this instead: keep this macro as is. add a new macro like so:
```
#define IGNORE_UNUSED_JNIPARAMETERS \
  (void) env; \
  (void) thiz
```

and add this as the first line to each of the functions? this way it is uniform how we handle it in all the functions (it doesn't really matter if we use these variables in these functinos or not since we know what we are ignoring here).

Reply:
It was between that approach and the one I have here, and I didn't have strong opinion either way. I'd be fine with adding an ignore parameter macro

Reply:
Done.